pcsx2/gui/Dialogs/GSDumpDialog.cpp

```diff
@@ -0,0 +1,830 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2020  PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "PrecompiledHeader.h"
+#include "App.h"
+#include "AppCommon.h"
+#include "MSWstuff.h"
+
+#include "Dialogs/ModalPopups.h"
+
+
+#include "Utilities/EmbeddedImage.h"
+#include "Resources/NoIcon.h"
+#include "GS.h"
+
+#include "PathDefs.h"
+#include "AppConfig.h"
+#include "Plugins.h"
+#include "GSFrame.h"
+#include "Counters.h"
+
+#include <wx/mstream.h>
+#include <wx/listctrl.h>
+#include <wx/filepicker.h>
+#include <wx/radiobut.h>
+#include <wx/button.h>
+#include <wx/treectrl.h>
+#include <wx/checkbox.h>
+#include <wx/dir.h>
+#include <wx/image.h>
+#include <wx/wfstream.h>
+#include <functional>
+
+namespace GSDump
+{
+	bool isRunning = false;
+}
+
+using namespace pxSizerFlags;
+
+// --------------------------------------------------------------------------------------
+//  GSDumpDialog  Implementation
+// --------------------------------------------------------------------------------------
+
+Dialogs::GSDumpDialog::GSDumpDialog(wxWindow* parent)
+	: wxDialogWithHelpers(parent, _("GSDumpGov"), pxDialogFlags())
```

Comment:
Why not GS Debugger or something similar?

Reply:
Because I forgot about it! WIll change it